src/translations/translations.en.js

```diff
@@ -660,6 +660,13 @@ export const translations = {
     return `${formattedValue} ${request} (${formatPercent(lang, percent)})`;
   },
   //#endregion
+  //#region cc-toast
+  'cc-toast.close': `Close`,
```

Comment:
@florian-sanders-cc do we need to improve this with some kind of `close this toas/notification`?